primitives/bridge/src/lib.rs

```diff
@@ -320,3 +323,30 @@ impl ConvertMessage for DoNothingConvertMessage {
         Err(ConvertMessageError::UnsupportedVersion)
     }
 }
+
+pub struct EthereumLocationsConverterFor<AccountId>(PhantomData<AccountId>);
```

Comment:
I see a slightly different implementation upstream, can you add a comment to why we don't use that one?

https://github.com/paritytech/polkadot-sdk/blob/711e6ff33373bc08b026446ce19b73920bfe068c/bridges/snowbridge/primitives/router/src/inbound/mod.rs#L467

Reply:
because that uses parents 1, and we are itself a solo-chain (not backed by any parent)